cookbook/form/direct_submit.rst

```diff
@@ -84,6 +84,13 @@ method, pass the submitted data directly to
 
         $form->get('firstName')->submit('Fabien');
 
+.. tip::
+
+    When submitting a form via a "PATCH" request, you may want to update only a few
+    submitted fields. To achieve this, you may pass an optional second boolean
+    parameter to ``submit()``. Passing ``false`` will remove any missing fields
```

Comment:
@HeahDude Shouldn't it be `$form->submit(…)` instead, as in the example above the tip?